### PR TITLE
Automation of downloading dependent .jars to "ESB_HOME/lib" directory

### PR DESCRIPTION
**Pull request by SangeethKarunaratne**
# Goals
> Provide user with suitable UI inorder to test the connectivity to his/her database when the required connection parameters are provided by the user. And also to download the 3rd party connectivity .jars or allow user to locate the required jar locally.

Providing the user a UI, details required for the jar to be downloaded and the data required for making connections to the database are collected.
After providing the details the user can check for the connectivity to the database and save the data in the UI which will then fill the columns in the property view of the DBLookupMediator / DBReportMediator.

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ConnectionObj.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
+package org.wso2.developerstudio.eclipse.gmf.esb.parts.forms;
```
Comment (dilee) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ConnectionObj.java]: Let's add the license header

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java [hunk 2]
```diff
@@ -2352,7 +2476,7 @@ public String getDescription() {
 	 * @see org.wso2.developerstudio.eclipse.gmf.esb.parts.DBLookupMediatorPropertiesEditionPart#setDescription(String newValue)
 	 * 
 	 */
-	public void setDescription(String newValue) {
+	public void  setDescription(String newValue) {
```
Comment (dilee) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]:
```suggestion
	public void setDescription(String newValue) {
```

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DependancyProvider.java [hunk 3]
```diff
@@ -0,0 +1,1220 @@
+package org.wso2.developerstudio.eclipse.gmf.esb.parts.forms;
```
Comment (dilee) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DependancyProvider.java]: Let's add the license header

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DependancyProvider.java [hunk 4]
```diff
@@ -0,0 +1,1220 @@
...
+import org.osgi.framework.BundleException;
+import org.w3c.dom.Attr;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.wso2.carbon.tools.converter.BundleGeneratorTool;
+import org.xml.sax.SAXException;
+
+public class DependancyProvider extends Dialog {
```
Comment (dilee) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DependancyProvider.java]: Typo: "Dependancy" should be "Depend**e**ncy"

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java [hunk 5]
```diff
@@ -2131,7 +2131,7 @@
 			public static String sourceProperty = "esb::EnrichMediator::Source::sourceProperty";
 			
 			// Start of user code for sourceXPath ElementEditor key
-			public static String sourceXPath = "esb::EnrichMediator::source::sourceXPath";
+			public static String sourceXPath = "esb::RuleMediator::source::sourceXPath";
```
Comment (praveennadarajah) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java]: Shall we remove unnecessary changes here?

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java [hunk 6]
```diff
@@ -1549,7 +1549,7 @@
 			public static String propertyValue = "esb::LogProperty::properties::propertyValue";
 			
 			// Start of user code for propertyExpression ElementEditor key
-			public static String propertyExpression = "esb::LogProperty::properties::propertyExpression";
+			public static String propertyExpression = "esb::ClassProperty::properties::propertyExpression";
```
Comment (praveennadarajah) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java]: Shall we remove unnecessary here?

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java [hunk 7]
```diff
@@ -3100,7 +3100,7 @@
 			public static String beanName = "esb::SpringMediator::properties::beanName";
 			
 			// Start of user code for configurationKey ElementEditor key
-			public static String configurationKey = "esb::SpringMediator::key::configurationKey";
+			public static String configurationKey = "esb::SmooksMediator::key::configurationKey";
```
Comment (praveennadarajah) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java]: Shall we remove unnecessary changes here?

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java [hunk 8]
```diff
@@ -211,12 +226,13 @@ public Composite createFigure(final Composite parent, final FormToolkit widgetFa
 	 * 
 	 */
 	public void createControls(final FormToolkit widgetFactory, Composite view) {
-	 // Start of user code for loading
```
Comment (praveennadarajah) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: Shall we add the comment bach?

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java [hunk 9]
```diff
@@ -211,12 +226,13 @@ public Composite createFigure(final Composite parent, final FormToolkit widgetFa
...
+	 
 	    CompositionSequence dBLookupMediatorStep = new BindingCompositionSequence(propertiesEditionComponent);
 	    
 	    CompositionStep connectionStep = dBLookupMediatorStep.addStep(EsbViewsRepository.DBLookupMediator.Connection.class);
-        connectionStep.addStep(EsbViewsRepository.DBLookupMediator.Connection.connectionType);
-        connectionStep.addStep(EsbViewsRepository.DBLookupMediator.Connection.connectionDsType);
+	    connectionStep.addStep(EsbViewsRepository.DBLookupMediator.Connection.connectionType);
```
Comment (praveennadarajah) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: Shall we fix the code formatting issues here?

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java [hunk 10]
```diff
@@ -340,10 +357,13 @@ public Composite addToPart(Composite parent, Object key) {
 				if (key == EsbViewsRepository.DBLookupMediator.Statements.sqlStatements) {
 					return createSqlStatementsTableComposition(widgetFactory, parent);
 				}
+				if (key == EsbViewsRepository.DBLookupMediator.Connection.databaseConfiguration) {
+					return createDbConfiguration(widgetFactory, parent);
+				}
 				return parent;
 			}
 		};
-		// End of user code
```
Comment (praveennadarajah) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: Shall we add the comment back here?

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java [hunk 11]
```diff
@@ -574,6 +594,110 @@ public void keyPressed(KeyEvent e) {
 		// End of user code
 		return parent;
 	}
+
```
Comment (praveennadarajah) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: Remove unnecessary lines here

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java [hunk 12]
```diff
@@ -1350,9 +1474,9 @@ protected Composite createConnectionDbTypeEMFComboViewer(FormToolkit widgetFacto
 			 * 	
 			 */
 			public void selectionChanged(SelectionChangedEvent event) {
-				if (propertiesEditionComponent != null)
-					propertiesEditionComponent.firePropertiesChanged(new PropertiesEditionEvent(DBLookupMediatorPropertiesEditionPartForm.this, EsbViewsRepository.DBLookupMediator.Connection.connectionDbType, PropertiesEditionEvent.COMMIT, PropertiesEditionEvent.SET, null, getConnectionDbType()));
-				fillDbConnectionDefaultValues();
+//				if (propertiesEditionComponent != null)
```
Comment (praveennadarajah) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: Shall we uncomment this code fragment if it is necessary or delete if it is unnecessary?

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBReportMediatorPropertiesEditionPartForm.java [hunk 13]
```diff
@@ -449,9 +570,9 @@ protected Composite createConnectionDbTypeEMFComboViewer(FormToolkit widgetFacto
 			 * 	
 			 */
 			public void selectionChanged(SelectionChangedEvent event) {
-				if (propertiesEditionComponent != null)
-					propertiesEditionComponent.firePropertiesChanged(new PropertiesEditionEvent(DBReportMediatorPropertiesEditionPartForm.this, EsbViewsRepository.DBReportMediator.Connection.connectionDbType, PropertiesEditionEvent.COMMIT, PropertiesEditionEvent.SET, null, getConnectionDbType()));
-				fillDbConnectionDefaultValues();
+//				if (propertiesEditionComponent != null)
```
Comment (praveennadarajah) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBReportMediatorPropertiesEditionPartForm.java]: Shall we fix the comment code fragment?

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBLookupMediatorPropertiesEditionComponent.java [hunk 14]
```diff
@@ -263,6 +269,7 @@ public boolean select(Viewer viewer, Object parentElement, Object element) {
 
 
 
+
```
Comment (dakshika) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBLookupMediatorPropertiesEditionComponent.java]: remove blank lines

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java [hunk 15]
```diff
@@ -574,6 +594,110 @@ public void keyPressed(KeyEvent e) {
 		// End of user code
 		return parent;
 	}
+	
+	
+	/**
```
Comment (dakshika) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: format the code properly

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java [hunk 16]
```diff
@@ -574,6 +594,110 @@ public void keyPressed(KeyEvent e) {
...
+	protected Composite createDbConfiguration(FormToolkit widgetFactory, Composite parent) {
+		Control propertyInitializeLabel = createDescription(parent,
+				EsbViewsRepository.DBLookupMediator.Connection.databaseConfiguration,
+				EsbMessages.DBLookupMediatorPropertiesEditionPart_DatabaseConfigurationLabel);
+		dependencyProvider = new Button(parent, SWT.NULL);
+
+		GridData dependencyProviderData = new GridData();
+		dependencyProvider.setText("Configure");
+		dependencyProviderData.grabExcessHorizontalSpace = false;
+		// dependencyProviderData.horizontalAlignment = SWT.BEGINNING;
```
Comment (dakshika) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: remove commented codes

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ConnectionObj.java (hunk 1 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ConnectionObj.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java (hunk 10 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java (hunk 2 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DependancyProvider.java (hunk 4 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DependancyProvider.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java (hunk 5 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java (hunk 9 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java (hunk 8 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java [hunk 17]
```diff
@@ -3100,7 +3100,7 @@
 			public static String beanName = "esb::SpringMediator::properties::beanName";
 			
 			// Start of user code for configurationKey ElementEditor key
-			public static String configurationKey = "esb::SpringMediator::key::configurationKey";
```
Comment (rosensilva) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java]: Shall we check the reason for changing Spiring mediator to Smooks mediaor

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DependancyProvider.java (hunk 3 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DependancyProvider.java]: License header added

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java (hunk 17 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java (hunk 7 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java (hunk 15 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java (hunk 6 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/EsbViewsRepository.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java (hunk 15 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java (hunk 11 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBLookupMediatorPropertiesEditionComponent.java (hunk 14 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBLookupMediatorPropertiesEditionComponent.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java (hunk 16 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBReportMediatorPropertiesEditionComponent.java [hunk 18]
```diff
@@ -655,7 +674,8 @@ public void updatePart(Notification msg) {
 			EsbPackage.eINSTANCE.getAbstractSqlExecutorMediator_PropertyValidationquery(),
 			EsbPackage.eINSTANCE.getAbstractSqlExecutorMediator_PropertyInitialsize(),
 			EsbPackage.eINSTANCE.getAbstractSqlExecutorMediator_SqlStatements(),
-			EsbPackage.eINSTANCE.getDBReportMediator_ConnectionUseTransaction()		);
+			EsbPackage.eINSTANCE.getDBReportMediator_ConnectionUseTransaction(),
+			EsbPackage.eINSTANCE.getMediator_Reverse()		);
```
Comment (rosensilva) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBReportMediatorPropertiesEditionComponent.java]: Why did we change this?

Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBReportMediatorPropertiesEditionComponent.java]: Due to code generation the code line had been added

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBReportMediatorPropertiesEditionPartForm.java (hunk 13 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBReportMediatorPropertiesEditionPartForm.java]: fixed

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java (hunk 12 above)
Comment (SangeethKarunaratne) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java]: fixed
